src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx

```diff
@@ -0,0 +1,153 @@
+
+import accountsConfiguredHealthStatus from 'images/accounts_icon_configured-health-status.svg'
+
+import solutionsAnomalyHealthStatus from 'images/solutions_icon_anomaly-health-status.webp'
+
+import solutionsMapEntity from 'images/solutions_screenshot-full_map-entity-example.webp'
+
+import solutionsFilterServiceMap from 'images/solutions_screenshot-full_filter-service-map.gif'
+
+import solutionsServiceMapsRightPane from 'images/solutions_screenshot-full_service-maps-right-pane.gif'
+
+A service map is a visual representations of your entire architecture from front-end to back-end, providing a comprehensive overview of the interconnected components that make up your system. This map dynamically displays the relationships between your applications, databases, hosts, servers, and out-of-process services ([web externals]([#externals])). 
+
+<img
+  title="Service map experience"
+alt="A screenshot of the service map experience. The screenshot shows a UI separated by white lines into three tiers. They are ordered vertically with UX as the top tier, Service as the second tier, and Infrastructure as the third tier."
+  src={SolutionsServiceMaps}
+/>
+
+<figcaption>
+  **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > APM > (choose an entity) > Service map**: Service map shows your dependencies and how they're performing.
+</figcaption>
+
+## Use service map [#using-service-maps]
+
+<Callout variant="tip">
+  Service map works with [distributed tracing](/docs/apm/distributed-tracing/getting-started/introduction-distributed-tracing) to connect relationships between entities. Service map is still functional if you have not enabled distributed tracing, but we recommend having distributed tracing enabled for all agents. This ensures a more consistent experience while using service maps.
+  </Callout>
+
+To access service map: 
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, then select either **APM & services**, **Mobile**, or **Browser**.
+2. Select an entity.
+3. On the left-hand side, click **Service map**.
+
+### Map from one entity [#map-from-one-entity]
+
+View how entities are related from a specific entity's point of view by hovoring your cursor over an entity and click **Map from this entity**. Service map will update from the perspective of that entity.
+
+<img
+  title="Open automap from service maps"
+  alt="A screenshot of the tooltip that appears when you hover your cursor over an entity."
+  src={solutionsMapEntity}
+  style={{ width: "70%" }}
+/>
+
+### Filter entities [#filter-map]
+
+You can modify the service map based on things like:
+
+* **Health status**: Remove the colored health outline around affected entities.
+* **Entity types**: Hide specific entities.
+* **Relationship depth**:
+  * **Depth 1**: (Default) Show all entities.
+  * **Depth 2**: Show degraded entities only.
+  * **Depth 3**: Show degraded entities only.
```

Comment:
@joandiazcapell What's the difference between depth 2 vs. 3? What's an example of what you'd see if you went from depth 1 to depth 2 to depth 3?

Reply:
By default, you are in relationship depth 1. This means you see all entities that have a direct relationship with the main entity. If you go to depth 2, you will go one hop ahead on relationships, you will also see direct relationships of these entities related to the main entity, and if you move to depth 3, the same, one hop more. The trick is, on depths 2 and 3 you only see entities with alerts. If we pain all entities you could end up with a huge unreadable map full of entities.
In this image, the main entity "account-provisional" has some relationships, if you move to depth 2 you will see all these new red entities on the top (inside a red box) that don't have a direct relationship from the main entity, they are two hops away from the main entity, they have the direct relationships with "Service - APm" which is not the main entity (depth 2)
<img width="2309" alt="Screenshot 2023-
11-07 at 09 45 29" src="https://github.com/newrelic/docs-website/assets/31217141/6d15e6ac-eb41-4856-a58e-2465de4ed2e4">